analyses/telomerase-activity-prediction/07_EXTEND-at-threshold.Rmd

```diff
@@ -0,0 +1,206 @@
+    by = "SampleID"
+  ) %>%
+  mutate(log2_fpkm = log2(fpkm+1))
+```
+
+## Re-analysis 1: Distribution of EXTEND scores
+
+Do trends in the overall distributions of EXTEND scores remain the same if performed only on tumors with high `tumor_fraction` values?
+
+
+```{r}
+ggplot(extend_df) + 
+  aes(x = cancer_group_display, 
+      y = extend_score) + 
+  geom_boxplot(fill = "gray60", 
+               # hide the outliers
+               outlier.size = NULL) + 
+  geom_jitter(width = 0.2, size = 0.5) +
+  facet_wrap(~extend_type, nrow = 2) +
+  theme(axis.text.x = element_text(angle = 30, hjust = 1, size = 6))
+
+```
+
+
+The plot above shows distributions before (top) and after (bottom) filtering, ordered by original EXTEND score median values.
+Previously, we observed from https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/b0fdf6f24fe9c021f22e8b7da348708f68877ee5/content/03.results.md?plain=1#L209:
+
+> We found aggressive tumors such as HGGs (DMGs and other high-grade gliomas) and MB had high `EXTEND` scores (**Figure {@fig:Fig4}D**), while low-grade lesions such as schwannomas, GNGs, DNETs, and other low-grade gliomas had among the lowest scores (**Table S3**).
+
+We see the order is broadly the same, so this means overall EXTEND score trends are largely the same.
+There are some minor differences and items to note:
+
+- First, there are no longer DIPGs for this plot (but these were excluded from the main text figure anyways since it's such a small group)
+- It's notable that pediatric meningiomas are _not_ highly aggressive, but have high scores (both without and with thresholding), so we may wish to clarify this further in the manuscript. also had high scores when analyzed previously so this result is not necessarily different but we might clarify it further.
+- Similarly, schwannomas are not actually terribly low in either set of distributions, so we may want to rethink this emphasis in the manuscript as well.
+
+
+
+## Re-analysis 2: Relationship to TERT and TERC expression
+
+Do we still see positive relationships between EXTEND scores and each of TERT and TERC expression?
+
+
+```{r}
+ggplot(fpkm_df) + 
+  aes(x = extend_score,
+      y = log2_fpkm) + 
+  geom_point() + 
+  geom_smooth(method = "lm") + 
+  facet_wrap(~gene) 
+```
+
+
+Let's calculate the actual correlations - 
+```{r}
+# subset dfs
+terc_df <- fpkm_df %>% filter(gene == "TERC")
+tert_df <- fpkm_df %>% filter(gene == "TERT")
+```
+
+
+TERC correlation:
+```{r}
+cor.test(terc_df$extend_score, terc_df$log2_fpkm)
+```
+
+TERT correlation:
+```{r}
+cor.test(tert_df$extend_score, tert_df$log2_fpkm)
+
+```
+
+
+Previously, we observed from https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/b0fdf6f24fe9c021f22e8b7da348708f68877ee5/content/03.results.md?plain=1#L208
+
+> As expected, `EXTEND` scores significantly correlated with _TERC_ (R = 0.619, p < 0.01) and _TERT_ (R = 0.491, p < 0.01) expression (**Figure {@fig:S5}B-C**).
```

Comment:
Any idea why these results are different? Maybe the lack of transformation before?

Reply:
The data going into the test is different (here, we're just testing a subset of the full stranded cohort), so I wouldn't expect the same values anyways? 
The data were previously transformed as well; this `mutate()` happens before modeling - https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/60791aac1eaa252137eaa1ddbe57bb4a7a1c51b1/figures/scripts/figS5-all-panels.R#L135

Reply:
Oh I see, I got confused - I thought the two lines of code were old results vs. new results and not two different genes.